egs/ami/s5b/local/nnet3/run_ivector_common.sh

```diff
@@ -17,8 +17,8 @@ train_set=train   # you might set this to e.g. train_cleaned.
 gmm=tri3          # This specifies a GMM-dir from the features of the type you're training the system on;
                   # it should contain alignments for 'train_set'.
 
-
 num_threads_ubm=32
+feat_type=lda
```

Comment:
let's call this ivector_transform_type so as not to confuse it with the feat_type used in other scripts.